fem/fe_coll.hpp

```diff
@@ -342,10 +342,17 @@ class NURBSFECollection : public FiniteElementCollection
    void Allocate(int Order);
    void Deallocate();
 
+   Array<int> Orders;
+
 public:
    explicit NURBSFECollection(int Order) { Allocate(Order); }
+   explicit NURBSFECollection(int px, int py, int pz);
+   explicit NURBSFECollection(Array<int>  &Orders);
+
+   int GetOrder() const { return ParallelepipedFE->GetOrder(); }
 
-   int GetOrder() const { return SegmentFE->GetOrder(); }
+   const Array<int>  &GetOrders() const {return Orders;}
+   //   void SetOrders(Array<int>  &Orders_) {Orders_.Copy(Orders);}
```

Comment:
Restore or delete?

Reply:
I think this routine can be deleted it is not used at the moment, and does not serve a clear purpose IMHO.